core/optaplanner-core-impl/src/main/java/org/optaplanner/core/impl/heuristic/selector/move/composite/UnionMoveSelectorFactory.java

```diff
@@ -52,20 +48,6 @@ public MoveSelector<Solution_> buildBaseMoveSelector(HeuristicConfigPolicy<Solut
             }
             selectorProbabilityWeightFactory = ConfigUtils.newInstance(config,
                     "selectorProbabilityWeightFactoryClass", config.getSelectorProbabilityWeightFactoryClass());
-        } else if (randomSelection) {
-            Map<MoveSelector<Solution_>, Double> fixedProbabilityWeightMap =
-                    new HashMap<>(config.getMoveSelectorList().size());
-            for (int i = 0; i < config.getMoveSelectorList().size(); i++) {
-                MoveSelectorConfig<?> innerMoveSelectorConfig = config.getMoveSelectorList().get(i);
-                MoveSelector<Solution_> moveSelector = moveSelectorList.get(i);
-                Double fixedProbabilityWeight = innerMoveSelectorConfig.getFixedProbabilityWeight();
```

Comment:
(hard) This is a regression.

There are two ways to create a selectorProbabilityWeightFactory, as a user:
- with a custom one, with config.getSelectorProbabilityWeightFactoryClass()
- with a syntactic sugar one, with moveSelectorConfig.getFixedProbabilityWeight()

The second one is being removed in this PR and we have users/customers using that. They will silently fall back on uniform distribution (which is probably why the integration tests don't fail).